### PR TITLE
chore(core): improve startup text

### PR DESCRIPTION
**Pull request by mtopolnik**
When using the CLI command `questdb` or `questdb.exe`: 

* print the full path to the QuestDB directory
* show the Web Console URL and ILP client connection string
* notify when HTTP or ILP is disabled, and show what config setting to change to enable it

## Current state (without this PR)

```plain
% questdb start -d db -n -f

  ___                  _   ____  ____
 / _ \ _   _  ___  ___| |_|  _ \| __ )
| | | | | | |/ _ \/ __| __| | | |  _ \
| |_| | |_| |  __/\__ \ |_| |_| | |_) |
 \__\_\\__,_|\___||___/\__|____/|____/
                        www.questdb.io

JAVA: /Users/mtopol/.sdkman/candidates/java/current/bin/java
```

## New state (after PR merged)

```plain
% questdb start -d ../dbroot-quest -n -f                                                          
JAVA: /Users/mtopol/.sdkman/candidates/java/current/bin/java

     ___                  _   ____  ____
    / _ \ _   _  ___  ___| |_|  _ \| __ )
   | | | | | | |/ _ \/ __| __| | | |  _ \
   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
    \__\_\\__,_|\___||___/\__|____/|____/
                        www.questdb.io

    Web Console URL                 ILP Client Connection String

    http://192.168.5.11:9000        http::addr=192.168.5.11:9000;
    http://127.0.0.1:9000           http::addr=127.0.0.1:9000;

QuestDB configuration files are in /Users/mtopol/dev/java/dbroot-quest/conf
```

If `line.http.enabled=false`:

```plain
% questdb start -d ../dbroot-quest -n -f
JAVA: /Users/mtopol/.sdkman/candidates/java/current/bin/java

     ___                  _   ____  ____
    / _ \ _   _  ___  ___| |_|  _ \| __ )
   | | | | | | |/ _ \/ __| __| | | |  _ \
   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
    \__\_\\__,_|\___||___/\__|____/|____/
                        www.questdb.io

    Web Console URL

    http://192.168.5.11:9000
    http://127.0.0.1:9000

InfluxDB Line Protocol is disabled for HTTP. Enable in server.conf: line.http.enabled=true
QuestDB configuration files are in /Users/mtopol/dev/java/dbroot-quest/conf
```

If `http.enabled=false`:

```plain
% questdb start -d ../dbroot-quest -n -f
JAVA: /Users/mtopol/.sdkman/candidates/java/current/bin/java

     ___                  _   ____  ____
    / _ \ _   _  ___  ___| |_|  _ \| __ )
   | | | | | | |/ _ \/ __| __| | | |  _ \
   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
    \__\_\\__,_|\___||___/\__|____/|____/
                        www.questdb.io

HTTP server is disabled. Enable in server.conf: http.enabled=true
QuestDB configuration files are in /Users/mtopol/dev/java/dbroot-quest/conf
```

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 38 / 39 (97.44%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/Bootstrap.java|35|36|97.22%|
|:large_blue_circle:|io/questdb/ServerMain.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/log/LogFactory.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 39 / 40 (97.50%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/Bootstrap.java|36|37|97.30%|
|:large_blue_circle:|io/questdb/ServerMain.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/log/LogFactory.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 39 / 40 (97.50%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/Bootstrap.java|35|36|97.22%|
|:large_blue_circle:|io/questdb/ServerMain.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/log/LogFactory.java|2|2|100.00%|

File: core/src/main/bin/questdb.sh [hunk 1]
```diff
@@ -202,10 +202,10 @@ function start {
     if [ "${QDB_CONTAINER_MODE}" != "" ]; then
         ${JAVA} ${JAVA_OPTS} -p ${JAVA_LIB} -m ${JAVA_MAIN} -d ${QDB_ROOT} ${QDB_OVERWRITE_PUBLIC} > ${QDB_LOG}/stdout-${DATE}.txt 2>&1
     elif [ "${QDB_DISABLE_HUP_HANDLER}" = "" ]; then
-        ${JAVA} ${JAVA_OPTS} -p ${JAVA_LIB} -m ${JAVA_MAIN} -d ${QDB_ROOT} ${QDB_OVERWRITE_PUBLIC} > ${QDB_LOG}/stdout-${DATE}.txt 2>&1 &
+        ${JAVA} ${JAVA_OPTS} -Dhello-to-stderr -p ${JAVA_LIB} -m ${JAVA_MAIN} -d ${QDB_ROOT} ${QDB_OVERWRITE_PUBLIC} > ${QDB_LOG}/stdout-${DATE}.txt &
```
Comment (bluestreak01) [inline core/src/main/bin/questdb.sh:205]: Ent version rust code is liable to produce critical messages into stderr. The stderr to stdout redirect was deliberate and should be kept. I realise this throws spanner in the works.

may be @eugenels or @amunra have ideas how to trap rust OOM erros and logs them to STDOUT?

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 58 / 63 (92.06%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/Bootstrap.java|46|51|90.20%|
|:large_blue_circle:|io/questdb/ServerMain.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/log/LogFactory.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 58 / 63 (92.06%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/Bootstrap.java|46|51|90.20%|
|:large_blue_circle:|io/questdb/ServerMain.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/log/LogFactory.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 58 / 63 (92.06%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/Bootstrap.java|46|51|90.20%|
|:large_blue_circle:|io/questdb/ServerMain.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/log/LogFactory.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 58 / 63 (92.06%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/Bootstrap.java|46|51|90.20%|
|:large_blue_circle:|io/questdb/ServerMain.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/log/LogFactory.java|2|2|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 58 / 63 (92.06%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/Bootstrap.java|46|51|90.20%|
|:large_blue_circle:|io/questdb/ServerMain.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/log/LogFactory.java|2|2|100.00%|

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**